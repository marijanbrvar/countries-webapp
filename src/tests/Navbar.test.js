import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router, Link } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router basename={process.env.PUBLIC_URL}>
        <nav>
          <div className="navbar">
            <Link
              to="/"
              className="brand-logo left"
            >
              <div>
                European Countries
              </div>
            </Link>
            <ul className="right">
              <li><i className="material-icons mx-1">mic</i></li>
              <li><i className="material-icons">settings</i></li>
            </ul>
          </div>
        </nav>
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
