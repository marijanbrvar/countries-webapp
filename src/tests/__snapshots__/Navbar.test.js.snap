// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav>
  <div
    className="navbar"
  >
    <a
      className="brand-logo left"
      href="/"
      onClick={[Function]}
    >
      <div>
        European Countries
      </div>
    </a>
    <ul
      className="right"
    >
      <li>
        <i
          className="material-icons mx-1"
        >
          mic
        </i>
      </li>
      <li>
        <i
          className="material-icons"
        >
          settings
        </i>
      </li>
    </ul>
  </div>
</nav>
`;

exports[`renders navbar`] = `
<nav>
  <div class="navbar">
    <a class="brand-logo left" href="/">
      <div>European Countries</div>
    </a>
    <ul class="right">
      <li><i class="material-icons mx-1">mic</i></li>
      <li><i class="material-icons">settings</i></li>
    </ul>
    </div>
  </nav>
`;
